reference_guide/internal_actions/enabling_internal.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Enabling the Internal Actions Menu
+---
+
+The Internal Actions menu is only visible if internal mode is enabled in IntelliJ IDEA.
+
+## Enabling Internal Mode
+There are multiple ways to enable internal mode, but the simplest is within IntelliJ IDEA:
+* Start IntelliJ IDEA.
+* From the main menu, select **Help | Edit Custom Properties**.
+This selection opens IntelliJ IDEA's `idea.properties` file.
+If a properties file does not exist, IntelliJ IDEA will prompt to create one.
+* Add the line shown below to the `idea.properties` file:
+![Enable Internal Mode](img/internal_mode_enable.png) 
```

Comment:
remove screenshot, adds no value IMHO
provide contents to be pasted in docs as text for c/p